docs/quickstart.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Guide
+
+This guide covers how you can quickly get started using Draft
+
+## Prerequisites
+
+The following prerequisites are required for successful use of Draft.
+
+1. A Kubernetes cluster and the `kubectl` CLI tool
+2. Installation of Helm on your Kubernetes cluster
+3. Installing and configuring Draft on your laptop
+
+### Install Kubernetes or have access to a cluster
+- If you don't already have a running Kubernetes cluster, consider downloading and installing [Minikube][] which can help you get a Kubernetes cluster running on your local machine
+- You can download and configure the `kubectl` binary using the directions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+### Install and configure Helm
+- Helm is a package manager for Kubernetes. It allows you to install and manage packages of Kubernetes manifests, more commonly known as "charts", on your Kubernetes cluster.
+- Download the Helm binary using [Homebrew](https://brew.sh/) via `brew install kubernetes-helm` or the [official releases page](https://github.com/kubernetes/helm/releases).
+- Once you have Helm on your machine and a running Kubernetes cluster, run the following command:
+```console
+$ helm init
+```
+
+### Install and configure Draft
```

Comment:
@bacongobbler - should we add Fish instructions for Draft?

Reply:
Yes; I'd love to.

Reply:
Maybe chocolatey as well? Especially for windows

Reply:
Theres no chocolatey package available for Draft. We're tracking that in another issue, but once that's available we can add that to the list as well.

Reply:
I will note however that GoFish works on Mac/Linux/Windows so it's something Windows users can try out.

@palma21 for reference: https://gofish.sh

Reply:
Yes I also use gofish but just stumbled on https://chocolatey.org/packages/draft hence the comment